packages/node_modules/@ciscospark/react-redux-spark/src/spark.js

```diff
@@ -24,71 +101,36 @@ export default function createSpark(accessToken, options = {}) {
           access_token: accessToken || credentials.access_token
         }
       },
-      config: {
-        appName: 'spark-widgets',
-        appVersion: process.env.REACT_CISCOSPARK_VERSION,
-        device: {
-          ephemeral: true
-        },
-        logger: {
-          level: options.logLevel || process.env.NODE_ENV === 'production' ? 'silent' : 'trace'
-        },
-        conversation: {
-          allowedInboundTags: {
-            'spark-mention': ['data-object-type', 'data-object-id', 'data-object-url'],
-            a: ['href'],
-            b: [],
-            blockquote: ['class'],
-            strong: [],
-            i: [],
-            em: [],
-            pre: [],
-            code: [],
-            br: [],
-            hr: [],
-            p: [],
-            ul: [],
-            ol: [],
-            li: [],
-            h1: [],
-            h2: [],
-            h3: []
-          },
-          allowedOutboundTags: {
-            'spark-mention': ['data-object-type', 'data-object-id', 'data-object-url'],
-            a: ['href'],
-            b: [],
-            blockquote: ['class'],
-            strong: [],
-            i: [],
-            em: [],
-            pre: [],
-            code: [],
-            br: [],
-            hr: [],
-            p: [],
-            ul: [],
-            ol: [],
-            li: [],
-            h1: [],
-            h2: [],
-            h3: []
-          }
-        },
-        // Added to help load blocking during decryption
-        encryption: {
-          kmsInitialTimeout: 10000
-        },
-        phone: {
-          enableExperimentalGroupCallingSupport: true
-        },
-        storage: {
-          unboundedAdapter: new LocalForageStoreAdapter('ciscospark-widgets')
-        }
-      }
+      config: defaultConfig(options)
     });
     window.ciscosparkInstance = ciscosparkInstance;
   }
 
-  return ciscosparkInstance;
+  return Promise.resolve(ciscosparkInstance);
+}
+
+/**
+ * Creates a spark instance with the jwt token generated
+ * by a guest issuer.
+ * https://developer.ciscospark.com/guest-issuer.html
+ *
+ * @param {string} jwt
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+export function createSparkJwt(jwt, options = {}) {
+  // Check if we already have a spark instance in memory
+  let {ciscosparkInstance} = window;
+
+  if (typeof ciscosparkInstance === 'object') {
+    return Promise.resolve(ciscosparkInstance);
+  }
+
+  ciscosparkInstance = new Spark({
+    config: defaultConfig(options)
+  });
+  window.ciscosparkInstance = ciscosparkInstance;
+  return ciscosparkInstance.authorization.requestAccessTokenFromJwt({jwt}).then(() => ciscosparkInstance);
```

Comment:
@ianwremmel this is the line I'd like to get your feedback on. Since we won't be passing an access token, is the instance that is returned after jwt auth the same thing that would be returned with an access token?

Reply:
yea, that'll work